lib/collection/widgets/first_pass_food_item_card.dart

```diff
@@ -65,6 +65,7 @@ class FirstPassFoodItemCard extends StatelessWidget {
             // TODO: Make a pop-up with a confirmation box
             // for deleting an item, this might need to be done in First Pass
             // Screen
+            // Or, make a three dot button that opens a window to delete
```

Comment:
I know that `Slidable` is used in the measurements and in the `Recipe` module, I just made that TODO because I think it is a matter of preference what item deletion method to use — to me, sliding feels like a very iOS thing to do, on Android it is often done either by long pressing or by some visible button, usually something subtle, yet visible, like a three-dot button in the corner of the item. I personally like when the user sees all the available actions, hence I'm a fan of buttons over long-presses or sliding, as these are not visible.

So tl;dr I implemented sliding to delete for now, but I'd be happy if we could run an AB test in the future with different options to see what the enumerators prefer the most 😊.